client/state/data-layer/wpcom/sites/automated-transfer/eligibility/index.js

```diff
@@ -44,10 +43,20 @@ const statusMapping = {
  *
  * @param {object} response API response data
  * @param {Array} response.errors List of { code, message } pairs describing issues
+ * @param {object} options object
  * @returns {Array} list of hold constants associated with issues listed in API response
+ *
  */
-const eligibilityHoldsFromApi = ( { errors = [] } ) =>
-	errors.map( ( { code } ) => get( statusMapping, code, '' ) ).filter( identity );
+export const eligibilityHoldsFromApi = ( { errors = [] }, options = {} ) =>
+	errors
+		.map( ( { code } ) => {
+			//differentiate on the client between a launched private site vs an unlaunched site
+			if ( options.sitePrivateUnlaunched && code === 'site_private' ) {
+				return eligibilityHolds.SITE_UNLAUNCHED;
```

Comment:
I wonder if we could update the eligibility endpoint to return a `site_unlaunched` hold rather that replacing it on the client.

Reply:
I considered it, but the number of resolvable holds by the customer is small and the fact this is going to be removed soon is why I opted for client only here.

If we had a ton of holds that were fixable, it'd make sense to centralize and add callbacks in server code. (For example providing the next api url to resolve a particular issue).